res/lang/programme/programme_en.txt

```diff
@@ -15,8 +15,8 @@ HIGH_AUDIENCE = High audience
 MOVIE_SPEED      = Speed
 MOVIE_CRITIC     = Critics
 MOVIE_BOXOFFICE  = Box-office
-MOVIE_TVAUDIENCE = TV-audience
-MOVIE_TOPICALITY = Topicality
+MOVIE_TVAUDIENCE = TV audience
+MOVIE_TOPICALITY = Freshness
```

Comment:
to discuss

Reply:
I resolve this conversation. Depending on the result. We'll just have to search for freshness.